app/models/recipe.rb

```diff
@@ -0,0 +1,9 @@
+class Recipe < ApplicationRecord
+  belongs_to :user
+  has_many :recipe_food
```

Comment:
- [ ] The same rule applys to Recipe as well. Kindly follow the first comment. Thank you :)

Reply:
@Rustamxon7 byt for reciepe i've already added this in destroy controller is it not correct?
![Uploading image.png…]()

Reply:
@rbhatt1999, It's great you added it! This option also works! However, it would be a great idea if you put the destroyer method in the model, as suggested. 

**Accept my apology, and please take my suggestion as optional :)**